index.html

```diff
@@ -217,5 +217,19 @@ <h2>
             </div>
         </section>
         </main>
+
+        <footer> 
+            <section class="contact">
+                <h2>Contact me</h2>
+              <p>I'm always interested in hearing about new projects,
+              so if you'd like to chat please get in touch.</p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
+                <input type="text" name="fullname" placeholder="Yeremias" maxlength="30" required>
+                <input type="email" name="email" placeholder="natajayanj14@gmail.com" required>
+                <textarea id="message" maxlength="500" placeholder="Write your message here" required></textarea>
```

Comment:
:red_circle: Kindly make sure you add the `name` attribute part of the textarea's attributes list. The attribute name is used to reference the form-data after submitting the form